ios/Surveys/Sources/Presentation/Modules/Home/HomeHeaderView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  HomeHeaderView.swift
+//  Surveys
+//
+//  Created by Su Ho on 03/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Kingfisher
+import SwiftUI
+
+struct HomeHeaderView: View {
+
+    private let title: String
+    private let imageURLString: String
+
+    var body: some View {
+        HStack(alignment: .center) {
+            VStack(
+                alignment: .leading,
+                spacing: 10.0
+            ) {
+                Text(title)
+                    .font(Typography.neuzeitSLTStdBookHeavy.font(size: 13.0))
+                    .foregroundColor(.white)
+                Text(Localize.homeHeaderToday())
+                    .font(Typography.neuzeitSLTStdBookHeavy.font(size: 34.0))
+                    .foregroundColor(.white)
+            }
+            Spacer()
+            KFImage(URL(string: imageURLString))
+                .resizable()
+                .frame(width: 36.0, height: 36.0)
+                .offset(y: 5.5)
+        }
+        .frame(height: 63.0)
```

Comment:
As AC #21 mentions that we have to:

> Display the white status bar

and we forgot to implement it.
So I think `.preferredColorScheme(.dark)` will solve that one. What do you think? 🙏 .

```suggestion
        }
        .preferredColorScheme(.dark)
        .frame(height: 63.0)
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/97/commits/53bd127fdd575af6b3e1af2f8515a6b72a02f44b